BUILDING.md

```diff
@@ -40,8 +40,15 @@ To build an image, run:
 cargo make
 ```
 
+This will build an image for the default variant, `aws-k8s-1.17`.
 All packages will be built in turn, and then compiled into an `img` file in the `build/` directory.
 
+To build an image for a different variant, run:
+
+```
+cargo make -e BUILDSYS_VARIANT=my-variant-here
+```
```

Comment:
`BUILDING.md` seems like a good place to hint at other controlling variables of interest, such as `BUILDSYS_ARCH`. Folks interested in building other variants may also want to build for another architecture.

Reply:
Agreed, but I'd prefer not to do that in the scope of this PR since generally documenting buildsys isn't quite the same thing as documenting the ECS variant...

Reply:
I see as a new level of detail and, with it, there's an opportunity to add a brief note on a closely related flag. 

That said, I understand holding off for another PR - it's your call.

Reply:
I'm not going to include it in this PR.